drivers/storage/portworx/status.go

```diff
@@ -109,7 +109,11 @@ func (p *portworx) updateStorageNodes(
 	// If cluster is running internal kvdb, get current bootstrap nodes
 	kvdbNodeMap := make(map[string]*kvdb_api.BootstrapEntry)
 	if cluster.Spec.Kvdb != nil && cluster.Spec.Kvdb.Internal {
-		strippedClusterName := strings.ToLower(configMapNameRegex.ReplaceAllString(cluster.Name, ""))
+		clusterID := cluster.Name
+		if cluster.Annotations != nil && cluster.Annotations[pxutil.AnnotationClusterID] != "" {
```

Comment:
we don't need to change here, the configMap name should always be valid here as long as cluster.Name is valid. I dont understand why we need configMapNameRegex.ReplaceAllString in the old code.

Reply:
yes you're right, cm name should be valid if cluster name is valid. but we originally have this logic here, to strip the cluster name, remove all special chars. I see we have this vendor with the same logic https://github.com/libopenstorage/operator/blob/master/vendor/github.com/portworx/kvdb/api/bootstrap/k8s/k8s.go probably it's just for consistency, the kvdb path uses the same format. So I retain this logic to avoid any regressions caused by changing the logic here.

Reply:
sounds good

Reply:
Yes, let's not change the names here because portworx creates the name. We just use the same regex as Portworx to match the configmap name.

Reply:
I mean what @pure-jliao has done is correct.